torch/nn/modules/rnn.py

```diff
@@ -507,6 +507,23 @@ class LSTM(RNNBase):
         All the weights and biases are initialized from :math:`\mathcal{U}(-\sqrt{k}, \sqrt{k})`
         where :math:`k = \frac{1}{\text{hidden\_size}}`
 
+    .. warning::
+        There are known deterministic issues for LSTM using cuDNN 7.6.5, 8.0 on CUDA 10.1 or later.
+        If you encounter this and need deterministic behavior,
+        a temporary workaround is to set environment variables.
+        See `cuDNN 8 Release Notes`_.
```

Comment:
This appears at an odd place since you were just introducing environment variables, but now there's a pointer to the cuDNN release notes?